pkg/koordlet/qosmanager/plugins/cpuevict/cpu_evict.go

```diff
@@ -142,21 +142,27 @@ func (c *cpuEvictor) calculateMilliRelease(thresholdConfig *slov1alpha1.Resource
 	// BECPULimit
 	avgBECPUMilliRealLimit, count03 := getBECPUMetric(metriccache.BEResouceAllocationRealLimit, querier, queryparam.Aggregate)
 
+	// CPU Satisfaction also considers the allocatable.
+	beCPUMilliAllocatable := c.getBEMilliAllocatable()
+	avgBECPUMilliLimit := math.Min(avgBECPUMilliRealLimit, beCPUMilliAllocatable)
```

Comment:
add an args for choosing which one to use, avgBECPUMilliRealLimit(default) or beCPUMilliAllocatable

Reply:
done. add a CPUEvictPolicy field in NodeSLO.